docs/nodes/vector/turbulence.rst

```diff
@@ -46,7 +46,7 @@ Inputs & Parameters
 Range table
 -----------
 Scalar values from turbulence node with size(n.verts)=64x64, step=0.05, octaves=3, amplitude=0.5, frequency=2.0, random seed=0.
-
+Plug a map range node in the scalar output and map it to the desired range (min=0, max=1) as in the image below.
 
 +----------------+----------------------+---------------------+----------------------+
 |  Noise Type    |       media          |  maximum            |   minimum            |
```

Comment:
probably, the average values between min and max, i used numpy.median() in italina we say media (latin words also)